docs/source/conf.py

```diff
@@ -120,7 +120,18 @@
             "icon": "fa-brands fa-github",
             # The type of image to be used (see below for details)
             "type": "fontawesome",
-        }
+        },
+        {
+            # Label for this link
```

Comment:
It's sufficient to have these comments in the first example (github), no need to duplicate them for every example